main.go

```diff
@@ -111,6 +111,7 @@ func router() *mux.Router {
 	router := mux.NewRouter()
 	// Unauthed pages
 	router.HandleFunc("/login", main.LoginHandler)
+  router.HandleFunc("/403", main.ForbiddenHandler)
```

Comment:
gofmt?

Reply:
Yep.. Was just waiting for the approval of this PR, to keep the diffs easy to notice without all the formatting changes.